src/pages/index.tsx

```diff
@@ -3,34 +3,75 @@ import React from 'react';
 import Layout from '../components/layout/layout';
 import SEO from '../components/seo';
 import { Grid, GridItem } from '../components/grid/grid';
+import { Typography } from '../components/typography/typography';
+import styled from 'styled-components';
+import { up, down } from '../components/breakpoint/breakpoint';
+
+const Title = styled.div`
+  margin-top: 108px;
+  margin-bottom: 280px;
+
+  ${up('md')} {
+    margin-top: 192px;
+    margin-bottom: 360px;
+  }
+`;
+
+const Subtitle = styled.div`
+  margin-top: 80px;
+  margin-bottom: 80px;
+
+  ${up('md')} {
+    margin-top: 160px;
+    margin-bottom: 120px;
+  }
+`;
+
+const DesktopOnlyBreak = styled.br`
+  ${down('md')} {
+    display: none;
+  }
+`;
 
 const IndexPage: React.FC = () => (
   <Layout>
     <SEO title="Home" />
     <Grid>
-      <GridItem md={2} />
-      <GridItem md={8}>
-        <h2>
-          <div>Full Stack</div>
-          <div>Digital Service</div>
-          <div>Agency</div>
-        </h2>
-        <p>We offer only what we are truly great at</p>
-        <div>
-          <div>Service</div>
-          <div>Full Stack</div>
-          <div>
-            Satellytes ist eine Digital-Agentur, die um große Unternehmen kreist
-            und ihnen bei der Transformation und Optimierung digitaler Services
-            und Interfaces hilft. Wir bieten „Full Stack“ an, also den gesamten
-            Prozess von Ideation bis zur Implementierung des letzten
-            performanten Funnels und der letzten Zeile wunderschönen Codes.
-          </div>
-          <button>All Services</button>
-        </div>
-        <p>We are into relationships, not one-night-stands.</p>
+      <GridItem xs={0} md={2} />
+      <GridItem xs={12} md={8}>
+        <Title>
+          <Typography variant="h1">
```

Comment:
What's the goal of "Typography" ? Only Headlines? any other tag? I don't know if this is really readable and scaleable. This could be an internal generalized tool but I would have expected something like:

```
<Title>ABC</Title>
<Subtitle>CDE</Subtitle>
```

Whatever you need to do in Title. But it's much more readable., don't you think?

Reply:
👍